indico/web/client/js/react/util/Responsive.jsx

```diff
@@ -0,0 +1,59 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Responsive from 'react-responsive';
+
+const DIMENSIONS = {
+  tablet: 768,
```

Comment:
How about using https://github.com/css-modules/icss#export (at least checking how easy it is to import those variables from `responsive.scss`)

Reply:
Good idea, I'll give it a try :+1:

Reply:
Not sure if it's worth using that just for this use case.